src/olympia/discovery/data.py

```diff
@@ -12,8 +12,8 @@ def __init__(self, *args, **kwargs):
 # At the moment the disco pane items are hardcoded in this file in the repos,
 # which allows us to integrate in our translation workflow easily.
 discopane_items = [
-    # Theme: Ice Flower
-    DiscoItem(addon_id=265447),
+    # Theme: Symphony of colors
```

Comment:
The github-issue writes "Colors" instead of "colors" - for now I always went with the copy that we get from Scott.